packages-common/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -0,0 +1,123 @@
+const { execSync } = require("child_process");
+const findFreePort = require("find-free-port");
+const { readFile, readdir } = require("fs").promises;
+const fetch = require("node-fetch");
+const { join } = require("path");
+const semverCompare = require("semver/functions/rcompare");
+const { cp, rm } = require("shelljs");
+
+main().catch(e => {
+    console.error(e);
+    process.exit(-1);
+});
+
+async function main() {
+    // verify that docker exists and complain otherwise
+    try {
+        execSync("docker info");
+    } catch (e) {
+        throw new Error("To run e2e test locally, make sure docker is running. Exiting now...");
+    }
+
+    const latestMendixVersion = await getLatestMendixVersion();
+    const packageConf = JSON.parse(await readFile("package.json"));
+    const teamServerProject = packageConf?.config?.testProjectId;
+    const widgetVersion = packageConf?.version;
+    const branch = packageConf?.config.e2eBranch || "trunk";
+
+    if (!teamServerProject) {
+        throw new Error(
+            "Currently e2e tests can only run against a teamserver project. Please provide the id of such project through config.testPorjectId in you package.json"
+        );
+    }
+    if (!process.env.TS_USERNAME || !process.env.TS_PASSWORD) {
+        throw new Error(
+            "Currently e2e tests can only run against a teamserver project. Please provide the credentials to access TeamServer through TS_USERNAME and TS_PASSWORD environment variables."
+        );
+    }
+    try {
+        const widgetMpk = (await readdir(`dist/${widgetVersion}`)).filter(el => /\.mpk$/.test(el)).length;
```

Comment:
🎄 we might be able to use `ls` from shelljs to verify this easier

Reply:
Fixed